cypress/support/step_definition/rooms/roomSteps.spec.js

```diff
@@ -3,13 +3,19 @@ import Rooms from "../../pages/rooms/pageRooms";
 
 const rooms = new Rooms();
 
-//Then('I should be redirected to the duplicated room with name containing {string}', (suffix) => {
-//verifyDuplicatedRoomNameContains(suffix);
-//})
+Then(
+	"I should be redirected to the duplicated room with name containing {string}",
+	(suffix) => {
```

Comment:
if we are not using this step then make sense to remove it. or are you planning to use it in future? in that case `roomNameSuffix` will make sense as a parameter name

Reply:
done.